src/coffee/sync/utils/product.coffee

```diff
@@ -452,16 +444,16 @@ class ProductUtils extends BaseUtils
       delete image._MATCH_CRITERIA
       action =
         action: 'addExternalImage'
-        variantId: variant.id
         image: image
+      @_setSkuOrVariantIdToUpdateAction(variant, action)
```

Comment:
Can we use the same pattern for naming everywhere? I would suggest using either `old_variant` or `new_variant` ?

Reply:
https://github.com/sphereio/sphere-node-sdk/pull/324/commits/48976579d71080c3d783e187e5a8db2b74ea3ffe